run-server.sh

```
@@ -0,0 +1,9 @@
+#!/bin/bash
+cd server
```

Shellcheck (0.10.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)